pyproject.toml

```
@@ -6,9 +6,9 @@ dependencies = [
     "craft-cli>=2.4.0",
     "craft-parts>=1.21.1",
     "craft-providers>=1.20.0,<2.0",
+    "platformdirs",
```

```suggestion
    "platformdirs>=2.1.0",
```

This version introduced what we use :-)

---

tox 4.11.3 requires `platformdirs>=3.10`, so I am going to set it to 3.10